packages/demo/useCryptKeeper.ts

```diff
@@ -216,23 +227,24 @@ export const useCryptKeeper = (): IUseCryptKeeperData => {
   }, [client]);
 
   const getConnectedIdentity = useCallback(async () => {
-    const payload = await client?.getConnectedIdentity();
-
-    if (!payload) {
-      return;
+    try {
+      const fetchedConnectedIdentityMetadata = await client?.getConnectedIdentity();
+      setConnectedIdentityMetadata(fetchedConnectedIdentityMetadata);
+      toast(
+        `Getting Identity Metadata Successfully! ${JSON.stringify(Object.entries(fetchedConnectedIdentityMetadata!))}`,
+        {
+          type: "success",
+        },
+      );
+    } catch (error) {
+      toast(`${error as string}`, {
```

Comment:
This is related to all cases in this file

```suggestion
      toast((error as Error).message, {
```